app/uk/gov/hmrc/economiccrimelevyaccount/views/PaymentsView.scala.html

```diff
@@ -95,15 +97,18 @@
     rowData.map(data => {
         val status = resolveStatus(data.paymentStatus)
         val paymentDate = ViewUtils.formatLocalDate(data.paymentDate)
+        val link = refundBaseUrl + "?amount=" + data.refundAmount
         Seq(TableRow(content = HtmlContent(paymentDate)),
             TableRow(content = HtmlContent(messages("account.viewPayments.table.cell.reference", data.chargeReference))),
             TableRow(content = HtmlContent(messages("account.viewPayments.table.cell.period", ViewUtils.formatLocalDate(data.fyFrom),
                 ViewUtils.formatLocalDate(data.fyTo))), classes = "govuk-table__cell"),
             TableRow(content = HtmlContent(ViewUtils.formatMoney(data.amount)), classes = "govuk-table__cell"),
-            TableRow(content = HtmlContent(messages("account.viewPayments.table.status", status._1, status._2)), classes = status._2)
+            TableRow(content = HtmlContent(messages("account.viewPayments.table.status", status._1, status._2)), classes = status._2),
+            TableRow(content = HtmlContent(s"<a href=${'"'}$link${'"'}>" + messages("account.viewPayments.table.paymentsHistory.cell.6") + "</a>"), classes = "govuk-table__cell"),
```

Comment:
We can follow the pattern established already for this, to avoid putting **A** tag hard coded in view.

See following as an example:

https://github.com/hmrc/economic-crime-levy-account-frontend/blob/91d5415ae7cc467cdf45204366ed7ac6fd3ec3c2/conf/messages.en#L39C3-L39C3

https://github.com/hmrc/economic-crime-levy-account-frontend/blob/91d5415ae7cc467cdf45204366ed7ac6fd3ec3c2/app/uk/gov/hmrc/economiccrimelevyaccount/views/ReturnsView.scala.html#L47